xyzspaces/spaces.py

```diff
@@ -1002,3 +1003,30 @@ def add_features_kml(
                 features_size=features_size,
                 chunk_size=chunk_size,
             )
+
+    def add_features_geobuf(
+        self, path: str, features_size: int = 2000, chunk_size: int = 1
+    ):
+        """
+        To upload data from geobuff file to a space
+        :param path: Path to geobuff file
+        :param features_size: An int representing a number of features to upload at
+            a time.
+        :param chunk_size: Number of chunks for each process to handle. The default value
+            is 1, for a large number of features please use `chunk_size` greater than 1.
+        """
+
+        with tempfile.NamedTemporaryFile() as temp:
```

Comment:
No need to create a temporary file, you can directly call the method `add_features`